crates/matrix-sdk/src/room/common.rs

```diff
@@ -301,7 +301,7 @@ impl Common {
     pub(crate) async fn request_members(&self) -> Result<Option<MembersResponse>> {
         let mut map = self.client.inner.members_request_locks.lock().await;
 
-        if let Some(mutex) = map.get(self.inner.room_id()).map(|m| m.clone()) {
+        if let Some(mutex) = map.get(self.inner.room_id()).cloned() {
```

Comment:
Can you rebase to merge this commit with the first one please?

Reply:
Yup, I ~always autosquash the fixup commits after review.